main.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, **kwargs):
         self.session = ClientSession(loop=self.loop)
         self.start_time = datetime.datetime.utcnow()
         self.clean_text = commands.clean_content(escape_markdown=True, fix_channel_mentions=True)
+        self.plus_one = "<:plus1:572280122330185743>"
```

Comment:
Hmm, i would probably make this a constant variable and not a class variable, like:
```python
PLUS_ONE = "<:plus1:572280122330185743>"
``` 
before the cog